handlers/stream.go

```diff
@@ -78,9 +83,22 @@ func (st Streaming) ProcessHTTP(w http.ResponseWriter, req *http.Request) {
 
 			heathcheckTimer.Reset(st.HeartbeatInterval * time.Second)
 			w.(http.Flusher).Flush()
+		case msg := <-subscription:
+			st.logger.InfoR(req, "User connected")
+			_, _ = w.Write([]byte(msg))
+			w.(http.Flusher).Flush()
+			if st.wg != nil {
+				st.wg.Done()
+			}
+		case <-req.Context().Done():
+			st.logger.InfoR(req, "User disconnecting")
```

Comment:
This event will only be received when the user has disconnected; should be "user disconnected"